lambdas/issue-credential/src/issue-credential-handler.ts

```diff
@@ -41,6 +43,7 @@ export class IssueCredentialHandler implements LambdaInterface {
       throw new Error(`Error saving questions to dynamoDb ${errorText}`);
     }
     const correlationId = answerResults.Item.correlationId;
+    logger.info("answerResults: " + JSON.stringify(answerResults));
```

Comment:
We probably don't want to log answer results. Same for the whole event above

Reply:
This was added for debugging forgot to remove